docs/oasis-node/metrics.md.tpl

```diff
@@ -0,0 +1,7 @@
+# List of metrics reported by oasis-node
+
```

Comment:
Refactor this file a bit to also include instructions on how to enable metrics reporting for the oasis-node etc. Then at the end include a list of metrics.

Also link to Tendermint metrics in a section called something like "Consensus backend-specific metrics", subsection "Tendermint".